spec/order_spec.rb

```diff
@@ -0,0 +1,28 @@
+require './lib/order.rb'
+
+describe Order do
+  before 'each order do' do
+    subject.start_order(:hawaiian_pizza, 3)
+  end
+
+  it 'can i choose an item an store it?' do
+    expect(subject.current_order).to eq [{hawaiian_pizza: 7, no_of_item: 3, item_total: 21}]
+  end
+  it 'can hold an order for more than one of a certain item' do
+    expect(subject.current_order).to eq [{hawaiian_pizza: 7, no_of_item: 3, item_total: 21}]
```

Comment:
Metrics/LineLength: Line is too long. [92/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)